frontend/packages/data-portal/app/hooks/useDownloadModalQueryParamState.ts

```diff
@@ -191,6 +216,7 @@ export function useDownloadModalQueryParamState() {
     [getPlausiblePayload, plausible, setDownloadParams],
   )
 
+  /** @deprecated */
   const setTomogramConfig = useCallback(
```

Comment:
nit: setting this one to `setTomogramConfigDeprecated` and the other one to `setTomogramConfig` would save some future work.